Samples/TextRendering/cpp-win32/DWriteCoreGallery.sln

```diff
@@ -21,9 +21,9 @@ Global
 		Release|x86 = Release|x86
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{D6EC0907-F515-4F61-93EC-B89E19FA58B4}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{D6EC0907-F515-4F61-93EC-B89E19FA58B4}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{D6EC0907-F515-4F61-93EC-B89E19FA58B4}.Debug|Any CPU.Deploy.0 = Debug|Any CPU
+		{D6EC0907-F515-4F61-93EC-B89E19FA58B4}.Debug|Any CPU.ActiveCfg = Debug|ARM64
```

Comment:
This seems a bit odd... why is the "Any CPU" platform setting/building "ARM64"?

Reply:
Weird.... I must have misclicked somewhere. I'll fix